### PR TITLE
Fixed Centcom's ID computers, added some new ID sprites for the strike teams.

### PR DESCRIPTION
**Pull request by DeityLink**
and did some background access stuff in preparation of the future map changes on centcom.

new_ids

### CODE REVIEW

**Comment by PJB3005** [comment]
:+1:

File: code/game/machinery/computer/card.dm [hunk 1]
```diff
@@ -91,7 +104,10 @@
 	if(!istype(id_card))
 		return ..()
 
-	if(!scan && access_change_ids in id_card.access)
+	if(!is_centcom() && !scan && access_change_ids in id_card.access)
```
Comment (ComicIronic) [inline code/game/machinery/computer/card.dm]: You have to wrap your ins.

**Comment by SkowronX** [comment]
ye buddy

**Comment by DeityLink** [comment]
'kay

**Comment by clusterfack** [comment]
Squash

**Pull request merged**